src/app.js

```diff
@@ -1,21 +1,31 @@
 'use strict';
 
-/**
- * Implement sum function:
- *
- * Function takes 2 numbers and returns their sum
- *
- * sum(1, 2) === 3
- * sum(1, 11) === 12
- *
- * @param {number} a
- * @param {number} b
- *
- * @return {number}
- */
-function sum(a, b) {
-  // write code here
-  return a + b;
-}
-
-module.exports = sum;
+const http = require('http');
+
+const server = http.createServer((req, res) => {
+  const parsedUrl = new URL(`http://localhost:${PORT}${req.url}`);
+
+  const parts = parsedUrl.pathname.split('/').filter(part => part !== '');
```

Comment:
could be just like this I suppose:
```suggestion
  const parts = parsedUrl.pathname.split('/').filter(part => part);
```